proposals/0024-opacity-micromaps.md

```diff
@@ -75,11 +74,11 @@ overall.  See the D3D12 flag definition for more details:
 `RayQuery` objects, used with [RayQuery::TraceRayInline()][rq-trace], 
 also need to be aware that OMMs may be in use.  The above `RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS` 
 doesn't apply for inline raytracing however. `RayQuery` objects are 
-independent of raytracing pipelines. For `RayQuery` the template for 
+independent of raytracing pipelines. For `RayQuery`, the template for 
 instantiating the object includes a new optional `RAYQUERY_FLAGS` parameter:
 
 ```
-template<uint StaticRayFlags, uint RayQueryFlags = RAYQUERY_FLAG_NONE>
```

Comment:
I think this change is unnecessary, and potentially worse.  The reason for "Static" in "StaticRayFlags" is to say that the portion of ray flags that are static can be specified here, since you normally also supply ray flags at the TraceRayInline call (it differentiates the subset of ray flags supplied here).  This is the only place for RayQuery object flags, so I don't think it needs the same differentiation.  Since they are template arguments, they must be constant, but I don't think it's necessary to start each name with "const" here.

Reply:
Ok, I changed the template param names back.